odoo/models.py

```diff
@@ -1058,6 +1058,8 @@ def load(self, fields, data):
         :type fields: list(str)
         :param data: row-major matrix of data to import
         :type data: list(list(str))
+        :param key_fields: list of fields that can be used to match records (TODO: confirm if context is preferred)
```

Comment:
This needs a much better explanation of the semantics because "match record" isn't really helpful.

From what I understand this is used to *find* existing records in order to update them (rather than create new, similar records), possibly when there is no known xid? Or does it replace xids-finding?